### PR TITLE
add cuDNN 3.0 library

### PR DESCRIPTION
**Pull request by thiell**
Add NVIDIA cuDNN 3.0 library that depends on CUDA 7.5.18

### CODE REVIEW

**Comment by kcgthb** [comment]
Yes

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/0735cf68cb5a74c61ac4 for a full test report.

File: easybuild/easyconfigs/c/cuDNN/cuDNN-3.0.eb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+easyblock = 'Tarball'
+name = 'cuDNN'
+version = '3.0'
+
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%(namelower)s-7.0-linux-x64-v%(version)s-prod.tgz']
```
Comment (boegel) [inline easybuild/easyconfigs/c/cuDNN/cuDNN-3.0.eb]: hmm, is this publicly available?

missing `source_urls`?

**Comment by thiell** [comment]
see #2882

**Pull request closed without merging**